File: frameworks/TypeScript/deno/src/_handlers/json.ts
import { dynDate, MIME_JSON, SERVER } from "../depends.ts";

export const headers = new Headers([
  ["server", SERVER],
  ["content-type", MIME_JSON],
]);

export default (_: Request): Promise<Response> => {
  const HELLO_OBJ = { message: "Hello, World!" };
  headers.set("date", dynDate());
  return Promise.resolve(new Response(JSON.stringify(HELLO_OBJ), { headers }));
};

File: frameworks/Dart/shelf/lib/src/json.dart
import 'dart:convert';
import 'package:shelf/shelf.dart';

const jsonHeader = {'Content-Type': 'application/json'};
const _message = {'message': 'Hello, World!'};

Response json(Request request) => Response.ok(
      jsonEncode(_message),
      headers: jsonHeader,
    );
